includes/class-wc-gateway-stripe.php

```diff
@@ -350,7 +350,18 @@ public function admin_scripts() {
 
 		$suffix = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
 
-		wp_enqueue_script( 'woocommerce_stripe_admin', plugins_url( 'assets/js/stripe-admin' . $suffix . '.js', WC_STRIPE_MAIN_FILE ), array(), WC_STRIPE_VERSION, true );
+		wp_register_script( 'woocommerce_stripe_admin', plugins_url( 'assets/js/stripe-admin' . $suffix . '.js', WC_STRIPE_MAIN_FILE ), array(), WC_STRIPE_VERSION, true );
+
+		$params = array(
+			'time'             => time(),
+			'i18n_out_of_sync' => wp_kses(
```

Comment:
It's a bit hard to understand what `i18n_out_of_sync` means, maybe something like `message_i18n_out_of_sync` would work better?

Reply:
That's a convention from WC core, it looks like – many examples in [this module](https://github.com/woocommerce/woocommerce/blob/master/includes/class-wc-frontend-scripts.php) (and used by this extension [here](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/87365188784e413bf2fd66bc504637122b823a4e/assets/js/stripe-payment-request.js#L592)). Do you think it's worth changing in this case regardless?